packages/config/src/config/build.js

```diff
@@ -120,5 +120,11 @@ export default () => ({
       /vue-ssr-(client|modern)-manifest.json/
     ]
   },
-  friendlyErrors: true
+  friendlyErrors: true,
+  warningFixFilters: [
+    // Hide warnings about plugins without a default export (#1179)
```

Comment:
Maybe we can move to add this filter to `warningFixFilter()` and always append? (as it doesn't make sense to be configurable)

Reply:
@pi0 So make it as empty array as default value, move back the current filter in `warningFixFilters` and append here ?

Like 

```js
warningIgnoreFilter() {
  const filters = [
    warn => warn.name === 'ModuleDependencyWarning' &&
        warn.message.includes(`export 'default'`) &&
        warn.message.includes('nuxt_plugin_'),
    ...this.buildContext.buildOptions.warningIgnoreFilters
  ]

  return warn => !filters.some(ignoreFilter => ignoreFilter(warn))
}
``` ?

Reply:
The proposed change looks good to me. About wording, yes please that would be great!